src/components/PostPageDetail/index.js

```diff
@@ -64,13 +62,15 @@ const PostPageDetailIdComponent = (props) => {
   const [loading, setLoading] = React.useState(true);
   const [, setReaction] = React.useState(false);
   const [textComment, setTextComment] = React.useState('');
-  const [typeComment] = React.useState('parent');
+  const [typeComment, setTypeComment] = React.useState('parent');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `typeComment` state variable is now settable, whereas it was previously a constant. This change allows for dynamic comment types, but ensure that all instances where `typeComment` is used have been updated to handle this change.
```diff
- const [typeComment] = React.useState('parent');
+ const [typeComment, setTypeComment] = React.useState('parent');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->